scripts/npm-stats.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
+import { JSDOM } from 'jsdom';
+
+async function getStatsForPackage(packageName) {
+  const { data } = await axios.get(
+    `https://www.npmjs.com/package/${packageName}?activeTab=versions`
+  );
+
+  const dom = new JSDOM(data);
+  const ul = dom.window.document.querySelector(
+    '.a0dff0b1 ~ .c495d29d ~ .a0dff0b1 ~ .c495d29d'
+  );
+  const versionLines = ul.children;
+  const res: any[] = [];
+
+  for (let i = 1; i < versionLines.length; i++) {
+    const line = versionLines.item(i);
+    const [version, _, downloads] = line.children;
+    res.push({
+      version: version.textContent,
+      downloads: parseInt(downloads.textContent.replace(/,/g, ''))
+    });
+  }
```

Comment:
```suggestion
 Array.from(versionLines).map(line => line.children).map(([version,_,dl]) => ({ version: version.textContent, downloads: parseInt(dl.textContent.replace(/,/g, '')) }))
```
Maybe some cleanup needed but the `.map()` chain seems nicer to me